recipe/meta.yaml

```diff
@@ -1,8 +1,6 @@
-{% set version = "0.11.7" %}
```

Comment:
Note: Still shows up in the diff as the version was bumped. Please see the line below for the version bump.